src/liboslexec/backendllvm.cpp

```diff
@@ -683,7 +691,11 @@ BackendLLVM::llvm_load_device_string (const Symbol& sym, bool follow)
     else if (userdata_index < 0) {
         // Handle non-varying variables
         OSL_DASSERT (sym.data() && "NULL data in non-varying string");
+#if OPTIX_VERSION < 70000
         val = getOrAllocateCUDAVariable (sym);
+#else
+        val = getOrAllocateCUDAVariable (sym);
```

Comment:
Same line as above?

Reply:
They used to differ at some point during my work on this.  :-)  I'll clean those up